<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DATVDemodGUI</class>
 <widget class="RollupWidget" name="DATVDemodGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>520</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>512</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>512</width>
    <height>520</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>DATV Demodulator</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>496</width>
     <height>250</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>496</width>
     <height>250</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>496</width>
     <height>250</height>
    </size>
   </property>
   <property name="title">
    <string>DATV Settings</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>222</width>
      <height>222</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="screenTVLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="DATVScreen" name="screenTV" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>220</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>220</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>261</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="cmbStandard">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>DVB-S</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DVB-S2</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cmbModulation">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>BPSK</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>QPSK</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8PSK</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16APSK</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>32APSK</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>64APSKe</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16QAM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>64QAM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>256QAM</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cmbFEC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1/2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2/3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3/4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5/6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7/8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4/5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8/9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9/10</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="chkFastlock">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>FAST LOCK</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkViterbi">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>VITERBI</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkHardMetric">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>HARD METRIC</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Symbols/s</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Bandwidth</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkHDLC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>HDLC</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkAllowDrift">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ALLOW DRIFT</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spiNotchFilters">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="maximum">
      <number>32</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Notch filter</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkResample">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>85</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>RESAMPLE</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="prgSynchro">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>190</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="lblStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>140</y>
       <width>21</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>R</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spiSymbolRate">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1024000000</number>
     </property>
     <property name="singleStep">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spiBandwidth">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1000</number>
     </property>
     <property name="maximum">
      <number>1024000000</number>
     </property>
     <property name="singleStep">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLabel" name="lblRate">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>61</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Buffer:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>496</width>
     <height>240</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>496</width>
     <height>240</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>496</width>
     <height>240</height>
    </size>
   </property>
   <property name="title">
    <string>VIDEO Stream</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>358</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="screenTVLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="DATVideoRender" name="screenTV_2" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>356</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>356</width>
         <height>200</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="lblState">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>131</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>  Video</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Full Screen</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblRead">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblReadStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RollupWidget</class>
   <extends>QWidget</extends>
   <header>gui/rollupwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DATVScreen</class>
   <extends>QWidget</extends>
   <header>datvscreen.h</header>
  </customwidget>
  <customwidget>
   <class>DATVideoRender</class>
   <extends>QWidget</extends>
   <header>datvideorender.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
